be/src/storage/lake/filenames.h

```diff
@@ -85,6 +85,10 @@ inline std::string random_segment_filename() {
     return name;
 }
 
+inline std::string schema_filename(int64_t schema_id) {
+    return fmt::format("SCHEMA_{:016X}", schema_id);
```

Comment:
why not use `xxx.schema`?

Reply:
I did not consider too much, it is just a way of naming.